views/repo.html

```diff
@@ -55,7 +60,12 @@ <h3 class="panel-title">Deployments by Month</h3>
                 <h3 class="panel-title">URL Hash</h3>
               </div>
               <div class="panel-body">
-                <input type="text" class="form-control" readonly="readonly" value="{{url_hash}}">
+                <div class="input-group">
+                  <input type="text" class="form-control" readonly="readonly" value="{{url_hash}}" />
```

Comment:
I have always been a fan of closing void tags even though its not required with HTML. 

http://blog.teamtreehouse.com/to-close-or-not-to-close-tags-in-html5